### PR TITLE
Assert warnings and errors from missing colon

### PR DESCRIPTION
**Pull request by botandrose**
See https://github.com/sass/libsass/issues/2407 for more details.

Is there anything I can do to improve this?

### CODE REVIEW

**Comment by xzyfer** [comment]
Thanks for this @botandrose. You'll need to add the following

- error-dart-sass

```
"WARNING on line 2, column 3 of spec/libsass-issues/issue_2407/input.sass: 
This selector doesn't have any properties and won't be rendered.
  margin-top 1px
  ^^^^^^^^^^^^^^

Error: expected selector.
  margin-top 1px
             ^
  spec/libsass-issues/issue_2407/input.sass 2:14  root stylesheet"
```

**Comment by botandrose** [comment]
@xzyfer Thanks for the help! I've added the file.

File: spec/libsass-todo-issues/issue_2407/error-dart-sass [hunk 1]
```diff
@@ -0,0 +1,9 @@
+"WARNING on line 2, column 3 of spec/libsass-issues/issue_2407/input.sass:
```
Comment (xzyfer) [inline spec/libsass-todo-issues/issue_2407/error-dart-sass]: Remove this "

File: spec/libsass-todo-issues/issue_2407/error-dart-sass [hunk 2]
```diff
@@ -0,0 +1,9 @@
+"WARNING on line 2, column 3 of spec/libsass-issues/issue_2407/input.sass: 
+This selector doesn't have any properties and won't be rendered.
+  margin-top 1px
+  ^^^^^^^^^^^^^^
+
+Error: expected selector.
+  margin-top 1px
+             ^
+  spec/libsass-issues/issue_2407/input.sass 2:14  root stylesheet"
```
Comment (xzyfer) [inline spec/libsass-todo-issues/issue_2407/error-dart-sass]: Remove this "

**Comment by nex3** [comment]
Closing as stale.

**Pull request closed without merging**